geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -352,4 +362,23 @@ private ConfigurationManager getConfigurationManager(CacheElement config) {
 
     return configurationManager;
   }
+
+  private static void setGroup(Object config, String group) {
+    if (config instanceof Groupable)
```

Comment:
Always use braces.
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used

Reply:
I would be happy to add the braces, but I'd like to understand how this slipped through our tooling.  Shouldn't formatting/style issues should be caught/fixed by automated process, not by human review?

Can you point me to where in the Geode wiki our code formatting standard is specified?  I'm curious if it has changed recently.  It looks like `etc/intellij-java-modified-google-style.xml` has not been modified in over two years.

My IntelliJ is set up to use `etc/intellij-java-modified-google-style.xml` as per https://github.com/apache/geode/blob/develop/BUILDING.md and I ran spotlessApply before committing.  If these steps are not sufficient to ensure compliant code formatting/style, would it be possible to update our developer documentation and/or spotless rules and/or IntelliJ style.xml to match?